src/content/en/2022/page-weight.md

```diff
@@ -4,19 +4,443 @@ title: Page Weight
+
+Older formats like PNG, JPEG, GIF enjoy their legacy of broader "historical" browser support. Performance-focused file type <a hreflang="en" href="https://developers.google.com/speed/webp">WebP</a> gained significant browser support and is now available to <a hreflang="en" href="https://caniuse.com/webp">97% of global users</a>.
+
+To delve into the findings and implications of image use on the web, refer to the [Media](./media) chapter.
+
+### JavaScript
+
+JavaScript is the most popular client-side scripting language on the web. We see that [98% of websites](./markup#fig-10) use it to create interactive online content—and <a hreflang="en" href="https://w3techs.com/technologies/details/cp-javascript">other sources agree it's that high</a>. While magnificent when used in moderation, the intoxicating allure of JavaScript can also lead to serious performance, search engine optimization, and user experience issues.
+
+Refer to [JavaScript](./javascript) chapter of the for detailed insights into the internet's favorite monkey paw.
+
+### Third-party services
+
+A page's weight is not limited to the assets hosted on the origin. Third-party resources requested by the page pile onto the weight in the form of analytics, chatbots, forms, embeds, analytics, A/B testing tools, and data collection.
+
+According to the [Third Parties](./third-parties) chapter, 94% of all websites on mobile devices use at least one third-party resource! Each of these contributes to the byte size of page weight.
+
+### Other assets
+
+The basic building blocks of the web have remained relatively unchanged for over 25 years. As the richness of web experiences increase, so does the use of fonts and videos.
+
+These increases are in step with the other file weight increases with a notable expectation to the 100th percentile.
+
+{{ figure_markup(
+  caption="The largest font use on mobile page.",
+  content="110 MB",
+  classes="big-number",
+  sheets_gid="1763112644",
+  sql_file="bytes_per_type.sql"
+)
+}}
+
+In 2021, the 100th percentile of mobile sites used 20,452 kilobytes of font files. In 2022, these outliers swelled to 110 megabytes. This 540% growth was not seen in the year of year comparison for desktop which sat at 66,257 kilobytes in 2021 and 68,285 in 2022.
+
+However, the 100th percentile—while fun to investigate—will always show the worst of the web. At At the 90th percentile, the mobile font weight was a less extreme—though still large—401 kilobytes.
+
+More insights into the typographical nature of the web can be found in the [Fonts](./2021/third-parties) chapter.
+
+## Page weight by the numbers
+
+No we know what we are primarily interested in when considering page weight, let's dive into the details.
```

Comment:
I added this into text for this section, as gave Requests and Bytes their own sections.